docs/whats-new.asciidoc

```diff
@@ -1,78 +1,113 @@
+Other versions: {security-guide-all}/8.1/whats-new.html[8.1] | {security-guide-all}/8.0/whats-new.html[8.0] | {security-guide-all}/7.17/whats-new.html[7.17] | {security-guide-all}/7.16/whats-new.html[7.16] | {security-guide-all}/7.15/whats-new.html[7.15] | {security-guide-all}/7.14/whats-new.html[7.14] | {security-guide-all}/7.13/whats-new.html[7.13] | {security-guide-all}/7.12/whats-new.html[7.12] | {security-guide-all}/7.11/whats-new.html[7.11] | {security-guide-all}/7.10/whats-new.html[7.10] |
 {security-guide-all}/7.9/whats-new.html[7.9]
 
 // NOTE: The notable-highlights tagged regions are re-used in the Installation and Upgrade Guide. Full URL links are required in tagged regions.
 // tag::notable-highlights[]
 
 [discrete]
-[[name-changes-8.1]]
-== Terminology changes
-
-The *Exceptions* page is now named *Exception lists*.
+[[features-8.2]]
+== New features
 
 [discrete]
-[[features-8.1]]
-== New features
+=== New landing pages added to the left navigation menu
 
-*Rules page enhancements*
+Several new landing pages were added to the navigation menu in 8.2:
 
-The following applies to the *Rules* and *Rules monitoring* tabs.
+The *Getting started* page provides guidance on adding data to your environment. When new users install {elastic-sec}, this page is now the default view.
 
-As an optional technical preview, you can now {security-guide}/alerts-ui-monitor.html[sort these tables] by any column. To enable the technical preview, switch the *Technical preview* toggle on. If you experience performance issues, you can turn this setting off.
+[role="screenshot"]
+image::whats-new/images/8.2/getting-started.png[Getting started landing page]
+
+The {security-guide}/users-page.html[*Users page*] provides an overview of user data to help you understand authentication and user behavior.
 
 [role="screenshot"]
-image::whats-new/images/8.1/column-sort.gif[]
+image::whats-new/images/8.2/users-page.png[Users page]
+
+The {security-guide}/policies-page-ov.html[*Policies page*] allows you to view and manage your {endpoint-sec} integration policies from a single location.
+
+[role="screenshot"]
+image::whats-new/images/8.2/policies-page.png[]
+
+The {security-guide}/blocklist.html[*Blocklist*] page allows you to view, add, and manage the blocklist -- a list of specified applications that are blocked from running on hosts. You can also use the {security-guide}/blocklist-api.html[blocklist API] to manage blocked applications.
+
+[role="screenshot"]
+image::whats-new/images/8.2/blocklist-page.png[]
+
+[discrete]
+=== Session view tool shows Linux process executions (beta)
+
+{security-guide}/session-view.html[Session view] is a new tool that shows detailed information about Linux process executions in a chronological and hierarchal context. Use Session view to investigate alerts, user activity, and sessions on your Linux infrastructure.
```

Comment:
Based on the [docs themselves](https://www.elastic.co/guide/en/security/master/session-view.html), it looks like we're capitalizing "Session View."
```suggestion
{security-guide}/session-view.html[Session View] is a new tool that shows detailed information about Linux process executions in a chronological and hierarchal context. Use Session View to investigate alerts, user activity, and sessions on your Linux infrastructure.
```